worker/src/rebuild.rs

```diff
@@ -128,12 +131,33 @@ pub async fn rebuild(ctx: &Context<'_>, url: &str) -> Result<Rebuild> {
 
     // process result
     let output_path = out_dir.join(&filename);
+
+    info!("Generating signed link");
+
+
+    let signed_link = in_toto_run(
+        &format!("rebuild {}", filename.to_str().unwrap()),
+        None,
+        &[input_path.to_str().unwrap_or_else(|| "")],
```

Comment:
```suggestion
        &[input_path.to_str().ok_or_else(|| anyhow!("Input path contains invalid characters"))?],
```
This converts the `Option` into a `Result` that you can then use the `?` on :) If there are invalid characters the function would then just return that error.